src/api/app/models/branch_package.rb

```diff
@@ -150,7 +150,8 @@ def create_fork(tprj)
     opts = {}
     opts[:rebuild] = @rebuild_policy if @rebuild_policy
     opts[:block]   = @block_policy   if @block_policy
-    tprj.branch_to_repositories_from(params[:project], tpkg, opts)
+    sprj = Project.get_by_name(params[:project])
+    tprj.branch_to_repositories_from(sprj, tpkg, opts)
```

Comment:
that breaks branching from remote projects, I guess

Reply:
(yes, atm remote obs instances do not export <scmsync> tags, but that will change)

Reply:
What does the `scmsync` attribute have to do with branching repositories from a project?

Reply:
Nothing. But the fork command is for doing setup of another git fork based on an existing project. So it does not really copy repositories but reusing existing repositories in the new created project.

Reply:
and exactly this is what `branch_to_repositories_from` does. If `sprj` is a `Project` or a string (remote project).

Reply:
ah, right, I misread it, sorry.